index.js

```diff
@@ -78,11 +81,16 @@ function handleError(error) {
 }
 
 if (process.argv.length > 2) {
-  process.argv.slice(2).forEach(function(arg, index) {
-    readAuth(function(err, auth) {
-      if(err) handleError(err);
+  readAuth(function(err, auth) {
+    if(err) handleError(err);
```

Comment:
When `err` is present, there's no need to execute the next lines. I know you're exiting the program on `handleErr` but you should add a `return` statement here. This is a common pattern: returning earlier from a function, if something went wrong (sad path).